examples/protobufjs/defs.bzl

```diff
@@ -0,0 +1,102 @@
+"Illustrate how to wrap pbjs and pbts from protobufjs"
+
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+
+# TODO switch to protobufjs-cli when its published
+# https://github.com/protobufjs/protobuf.js/commit/da34f43ccd51ad97017e139f137521782f5ef119
+load("@npm//protobufjs:index.bzl", "pbjs", "pbts")
+load("@rules_proto//proto:defs.bzl", "ProtoInfo")
+
+# these deps are needed even tho they are not declared as dependencies of
```

Comment:
This comment needs a little bit of love :)

Reply:
kk